sklego/preprocessing.py

```diff
@@ -174,6 +174,10 @@ class ColumnSelector(BaseEstimator, TransformerMixin):
     """
 
     def __init__(self, columns: list):
+        # if the columns parameter is not a list, make it into a list
```

Comment:
The following if statement should be moved from fit to init method
   ` if not isinstance(self.columns, list):`
    `     self.columns = [self.columns]`

this is done for two reasons first to make the code consisent with
other similar classes(in this case ColumnDroper class). And second
because this checking is suitable during class instantiation and
has nothing specific to fit method.